string locking_node
---
bool lock_acquired
uint32 counter
